### DISCUSSION TITLE
Problem with 3.1.0 release

### DISCUSSION THREAD

**Discussion by GostCoffeeRoasters** [artisan-roaster-scope/artisan#1766] [General]
I have Mill City Roasters 2kg Phidget 4-input 1048_0 and Im not getting any reading.
I selected Config, Machine, Mill City Roaster , Phidget

**Comment by MAKOMO** [comment]
Did you install the Phidget Driver for your OS?

**Reply by GostCoffeeRoasters** [reply]
Im on Windows

**Reply by GostCoffeeRoasters** [reply]
Windows 10

**Reply by MAKOMO** [reply]
So did you install the Phidgets Driver from phidgets.com for Windows? Do you see the Phidget modules and their data in the Phidgets Control Panel (part of the driver package)?

**Reply by GostCoffeeRoasters** [reply]
I was using the older version of Artisan already with that Phidget. Should I download the drivers again?

**Reply by MAKOMO** [reply]
Artisan never came with a Phidget driver. You need to install the latest Phidget driver from https://phidgets.com/ and also the latest Artisan version, which is v3.1 currently.

**Reply by GostCoffeeRoasters** [reply]
I did and it actually it got worst now lol Im missing all temperature reading on the side. Im attaching screen shot so you can see what I see
![Screenshot 2024-12-20 100000](https://github.com/user-attachments/assets/b1eb31d4-c1b5-4a91-8c6b-19887ba07f42)

**Reply by poundy** [reply]
"missing all temperature reading on the side" means YOUR settings are set to not display them!  But from that it seems that the phidget driver has now addressed the lack of data, and other issues are just minor configuration. Do you have a saved copy of your settings from prior? 
From the VIEW menu, select READINGS. That should bring in your readings on the right hand side.

**Comment by GostCoffeeRoasters** [comment]
Here is what my screen use to look like before the new version of Artisan 3.1.0
![what the screen use to look like](https://github.com/user-attachments/assets/cc9042c2-7c21-4d02-955b-8448c0847c4d)

**Comment by GostCoffeeRoasters** [comment]
This is what the screen looks now
![Screenshot 2024-12-20 100000](https://github.com/user-attachments/assets/ba7a28ea-4bf6-4189-b3ed-747e331dddde)

**Reply by MAKOMO** [reply]
Looks to me like you have a hardware issue. The ET signal renders the BT and the ET signal is disconnected. Further it seems that this one temperature you get contains a huge amount of noise compared to your previous recordings. Maybe that TC is internally broken.

**Reply by GostCoffeeRoasters** [reply]
It was working perfectly fine the day of when I downloaded this new version and after this version everything when crazy. I haven't save my previous setting and now for days I been trying different combinations of setting with out any luck. Can you point me to someone who knows how to set this program up to work with Mill City Roaster?

**Reply by poundy** [reply]
Please stop replying in email and just use github on the web, it keeps the conversation much more logical if you keep these "threads" to keep track of activity. 

Anyone who knows how to set this up? Well, that would be you... We haven't had a lot of MCR owners share that they are using Artisan, to date. As far as I am aware, MCR did not sponsor the integration effort to add their roaster to the app. Not many others would own the roaster, we haven't had that many discussions about them in the past. The audience is pretty small... but having said that, these roasters, and Artisan are not rocket science, they are logical machines that can be understood and things worked on to "correct" problems

So, from here, why don't you try rolling back to the older version you installed and recover your settings? Why don't you tell us if the display on the side "fix" that I proposed above was successful? Why don't you share your current settings with us? Why don't you reset the Artisan settings again and go through the setup again, and see if that has an impact? 

Ultimately, you're in the driving seat here. You have some software that works, it clearly shows what Makomo suggests is a hardware issue, but only you can help eliminate that.  This is community sponsored software, so time for you to show some sweat equity.

**Comment by GostCoffeeRoasters** [comment]
This is my 2kg Mill City panel pictures 
![20241220_095249](https://github.com/user-attachments/assets/d1c5e93b-1a21-452f-ac01-3074f962bb91)
![20241220_095253](https://github.com/user-attachments/assets/dddec673-5213-40c7-bb4b-5567d5e6db76)

**Comment by GostCoffeeRoasters** [comment]
Before the update everything worked fine.

**Reply by MAKOMO** [reply]
Updating Artisan does not modify any settings. Artisan is back and forward compatible. Did you know that you can toggle visibility of the LCD readings on the side per state (OFF/ON/RECORDING) using menu View >> Readings? Did you know that you can toggle visibility of curves by a click on the corresponding LCD? So maybe this helps you to show the readings on the right side and toggle that missing ET curve back on.

Regarding the dramatic noise I see on your BT curve (and amplified in your RoR curve): this can come for the very badly shielded motors of that machine or using a long low-quality USB wire without Ferrite Beads at both ends, or simply broken TCs which produce ground noise. A good read is [Addressing Electromagnetic Inferences With Phidgets](https://www.phidgets.com/docs21/Addressing_Electromagnetic_Interference_with_Phidgets)
